pkg/engine/k8s/policy/deepcopy.extend.go

```diff
@@ -0,0 +1,52 @@
+package policy
```

Comment:
To discuss the file name: I wanted to set `zz_generated.deepcopy.extend.go`, but it may be confusing that it's generated. No strong opinion. WDYT?